tests/fixtures/editions.py

```diff
@@ -0,0 +1,15 @@
+import json
+import pathlib
+import pytest
+
+@pytest.fixture
```

Comment:
this wont work as the pytest.fixture decorator is already on the thing that's calling this. You can remove this.